NumLib/DOF/DOFTableUtil.cpp

```diff
@@ -44,6 +44,42 @@ std::vector<GlobalIndexType> getIndices(
     return indices;
 }
 
+NumLib::LocalToGlobalIndexMap::RowColumnIndices getRowColumnIndices(
```

Comment:
Indeed these functions are very similar. The important difference is that the `getRowColumnIndices` does not create the result vector `indices` again and again for every call of the function. Anyway, in this PR I only want to move existing code such that I can use it a second time at another place.